content/docs/csidriver/installation/operator/powermax.md

```diff
@@ -120,9 +132,16 @@ Create a secret named powermax-certs in the namespace where the CSI PowerMax dri
    | X_CSI_IG_NODENAME_TEMPLATE | Provide a template for the CSI driver to use while creating the Host/IG on the array for the nodes in the cluster. It is of the format a-b-c-%foo%-xyz where foo will be replaced by host name of each node in the cluster. | No | - |
    | X_CSI_POWERMAX_DRIVER_NAME | Set custom CSI driver name. For more details on this feature see the related [documentation](../../../features/powermax/#custom-driver-name) | No | - |
    | X_CSI_HEALTH_MONITOR_ENABLED | Enable/Disable health monitor of CSI volumes from Controller and Node plugin. Provides details of volume status, usage and volume condition. As a prerequisite, external-health-monitor sidecar section should be uncommented in samples which would install the sidecar | No | false |
+   | X_CSI_VSPHERE_ENABLED | Enable VMware virtualized environment support via RDM | No | false |
+   | X_CSI_VSPHERE_PORTGROUP | Existing portGroup that driver will use for vSphere | Yes | "" |
+   | X_CSI_VSPHERE_HOSTGROUP | Existing host group that driver will use for vSphere | Yes | "" |
+   | X_CSI_VCenter_HOST | URL/endpoint of the vCenter where all the ESX are present | Yes | "" |
```

Comment:
Is it normal the two variables are not all in UPPERCASE like others ?

Reply:
yes it has to be, will be addressed in the repos and changes will be reflected here post the same